metrics/src/lib.rs

```diff
@@ -71,7 +71,10 @@ impl Recorder for CombinedRecorder {
 
 #[cfg(feature = "prometheus")]
 pub fn initialize() -> Option<tokio::task::JoinHandle<()>> {
-    let prometheus_builder = metrics_exporter_prometheus::PrometheusBuilder::new();
+    let prometheus_builder = metrics_exporter_prometheus::PrometheusBuilder::new().set_buckets(&[
```

Comment:
Note: we may need to adjust these buckets but they seem to work quite will in my local tests.